web/src/features/observers/components/ObserverDetails/ObserverDetails.tsx

```diff
@@ -26,45 +27,56 @@ export default function ObserverDetails() {
-              <p className='text-gray-900 font-normal'>{observer.phoneNumber}</p>
-            </div>
-            <div className='flex flex-col gap-1'>
-              <p className='text-gray-700 font-bold'>Tags</p>
-              <p className='text-gray-900 font-normal'>N/A</p>
-            </div>
-         
-            <div className='flex flex-col gap-1'>
-              <p className='text-gray-700 font-bold'>Status</p>
-              <Badge className={'badge-' + observer.status}>{observer.status}</Badge>
-            </div>
-          </CardContent>
-        </Card>
+        <Tabs defaultValue='details'>
+          <TabsList className='grid grid-cols-2 bg-gray-200 w-[400px] mb-4'>
+            <TabsTrigger value='details'>Observer details</TabsTrigger>
+            <TabsTrigger value='elections'>Monitored elections</TabsTrigger>
+          </TabsList>
+          <TabsContent value='details'>
+            <Card className='w-[800px] pt-0'>
+              <CardHeader className='flex flex-column gap-2'>
+                <div className='flex flex-row justify-between items-center'>
+                  <CardTitle className='text-xl'>Observers details</CardTitle>
+                  <Button onClick={navigateToEdit} variant='ghost-primary'>
+                    <PencilIcon className='w-[18px] mr-2 text-purple-900' />
+                    <span className='text-base text-purple-900'>Edit</span>
+                  </Button>
+                </div>
+                <Separator />
+              </CardHeader>
+              <CardContent className='flex flex-col gap-6 items-baseline'>
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>First name</p>
+                  <p className='text-gray-900 font-normal'>{observer.firstName}</p>
+                </div>
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>Last name</p>
+                  <p className='text-gray-900 font-normal'>{observer.lastName}</p>
+                </div>
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>Email</p>
+                  <p className='text-gray-900 font-normal'>{observer.email}</p>
+                </div>
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>Phone</p>
+                  <p className='text-gray-900 font-normal'>{observer.phoneNumber}</p>
+                </div>
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>Tags</p>
+                  <p className='text-gray-900 font-normal'>N/A</p>
+                </div>
+
+                <div className='flex flex-col gap-1'>
+                  <p className='text-gray-700 font-bold'>Status</p>
+                  <Badge className={'badge-' + observer.status}>{observer.status}</Badge>
```

Comment:
create a component called `ObserverStatusBadge` similar to `src\components\ElectionRoundStatusBadge\ElectionRoundStatusBadge.tsx`